openBCISample.js

```diff
@@ -115,7 +148,7 @@ module.exports = {
         var intBuf = new Buffer(3); // 3 bytes for 24 bits
         intBuf.fill(0); // Fill the buffer with 0s
 
-        var temp = float / SCALE_FACTOR_CHANNEL; // Convert to counts
+        var temp = float / ( ADS1299_VREF / 24 / (Math.pow(2,23) - 1)); // Convert to counts
```

Comment:
Because i had to get rid of that constant because we now support different channel gains, see how `parseRawPacket` in the same file now takes `channelSettingsArray` which has the gain for each channel. `floatTo3ByteBuffer` is only for the simulator which only supports gain of 24... Does this make sense?